SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -343,11 +353,15 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
                 }
             }
         }
+        
+        self.incomingPushHandled()
     }
     
     func cancelledCallInviteReceived(_ cancelledCallInvite: TVOCancelledCallInvite) {
         NSLog("cancelledCallInviteCanceled:")
         
+        self.incomingPushHandled()
```

Comment:
should we call this at the end of the method (after executing other things in `cancelledCallInviteReceived`) ?